package.json

```diff
@@ -21,61 +35,42 @@
     }
   ],
   "files": [
-    "dist/",
-    "src/",
-    "postinstall.js"
+    "src/"
```

Comment:
I'd still like to export the minified version. Someone could npm install, then host the file on their own cdn or something. So I'd like to keep `dist/` here.